index.html

```diff
@@ -1,296 +1,360 @@
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
-  <link rel="stylesheet" href="styles/main.css">
-  <title>JT's Portfolio</title>
-
-</head>
-<body>
-  <div class="grid-container">
-    <header>
-      <div class="name-logo">
-        <a id="name-logo" href="#">Jud&aacute; Teixeira</a>
-      </div>
-      
-      
-      <!-- Desktop menu -->
-      <nav class="menu-desktop">
-        <a class="menu-hamb" href="#">
-          <img src="icons/menu.png" alt="Menu">
-        </a>
-        <ul class="menu-desktop-item">
-          <li><a href="#Portfolio">Portfolio</a></li>
-          <li><a href="#About">About</a></li>
-          <li><a href="#Contact">Contact</a></li>
-        </ul>
-      </nav>
-    </header>
-
-    <section class="main" id="bg-main">
-      <div class="main-elements">
-        <div class="content">
-         <h1>I'm Jud&aacute; <br>
-             Glad to see you!</h1>
-         <p>
-          I'm a software developer! I can help you 
-          build a product, feature or website Look 
-          through some of my work and 
-          experience! If you like what you see and 
-          have a project you need coded, don't 
-          hesitate to contact me.
-         </p>
-        
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Poppins"
+    />
+    <link rel="stylesheet" href="styles/main.css" />
+    <title>JT's Portfolio</title>
+  </head>
+  <body>
+    <div class="grid-container">
+      <header>
+        <div class="name-logo">
+          <a id="name-logo" href="#">Jud&aacute; Teixeira</a>
         </div>
-        <div class="socials">
-          <h2>LET'S CONNECT</h2>
-          <ul class="icon-bar">
-            <li><a href="#">
-              <img src="icons/twitter.png" alt="Twitter">
-            </a></li>
-            <li><a href="#">
-              <img src="icons/linkedin.png" alt="Linkedin">
-            </a></li>
-            <li><a href="#">
-              <img src="icons/facebook.png" alt="Facebook">
-            </a></li>
-            <li><a href="#">
-              <img src="icons/github.png" alt="Github">
-            </a></li>
-            <li><a href="#">
-              <img src="icons/instagram.png" alt="Instagram">
-            </a></li>
+        <nav class="menu-mobile-container hide">
+          <a class="menu-x" href="#">
+            <div>X</div>
+          </a>
+          <ul class="menu-mobile-list">
+            <li class="portfolio-btn"><a href="#Portfolio">Portfolio</a></li>
+            <li class="about-btn"><a href="#About">About</a></li>
+            <li class="contact-btn"><a href="#Contact">Contact</a></li>
           </ul>
-        </div>
-      </div>
-    </section> <!-- End of MAIN -->
+        </nav>
```

Comment:
- [ ] I think you missed out on the colour for the mobile menu popup. The given design colour is: `#6070FF;`

Your Design | Template Design  |
:---------------:|:------------
![a1](https://user-images.githubusercontent.com/39523312/156059750-f3d7d04b-b4d7-48ee-b81c-853e1ea5b2bf.png)|![a 2](https://user-images.githubusercontent.com/39523312/156059757-6748adbf-5d1a-4ce8-a7c1-78280e72901f.png)|

Reply:
Done, thank you :-)